.github/workflows/ci.yml

```diff
@@ -157,7 +157,9 @@ jobs:
         gem install bundler:2.2.9
         bundle config path vendor/bundle
         bundle update actionview activemodel activesupport railties
+        bundle exec rake docs:preview
         bundle exec rake
+        rm -rf demo/test/components/previews/primer/docs/
```

Comment:
do we have to remove them? I think this is ephemeral 🤔

Reply:
hrmmm I actually wasn't sure about this but looking into it a bit more, it seems we don't need this line:

```
each GitHub-hosted runner is always a clean isolated virtual machine, and it is 
destroyed at the end of the job execution.
```

https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#self-hosted-runner-security-with-public-repositories